apps/viewer/src/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { useMemo } from 'react';
+
+export interface URLParameters {
+  streamName: string;
+  streamAccountId: string;
+  isMultiviewEnabled: boolean;
+}
+
+export const useURLParameters = (): URLParameters => {
+  return useMemo(() => {
+    const href = new URL(window.location.href);
+
+    const streamName = href.searchParams.get('streamName') ?? import.meta.env.VITE_RTS_STREAM_NAME;
+    const streamAccountId = href.searchParams.get('streamAccountId') ?? import.meta.env.VITE_RTS_ACCOUNT_ID;
+    const isMultiviewEnabled = href.searchParams.get('multisource')?.toLowerCase() === 'true';
```

Comment:
So having it this way implies that if the query param isn't set, then only the main source will be present. 

However, on the dashboard the behaviour is - 
```
assume multiview is enabled by default, unless explicitly disabled
```

Reply:
Perhaps the more appropriate check is this 
```suggestion
    const isMultiviewEnabled = href.searchParams.get('multisource')?.toLowerCase() !== 'false';
```

Reply:
Yeah, given the dashboard behaviour you mentioned above, this is definitely the right implementation
Addressed, thanks <3